butterfly-cli/src/main/java/com/paypal/butterfly/cli/ButterflyCliOption.java

```diff
@@ -27,6 +27,7 @@
     protected static final String CLI_OPTION_TEMPLATE_SHORTCUT = "s";
     protected static final String CLI_OPTION_UPGRADE_VERSION = "u";
     protected static final String CLI_OPTION_RESULT_FILE = "r";
+    protected static final String CLI_OPTION_IN_PLACE_TRANSFORM = "f";
```

Comment:
Please, rename this to `CLI_OPTION_MODIFY_ORIGINAL_FOLDER`. That is better aligned with `-f`.